package.json

```diff
@@ -153,11 +153,14 @@
     "react-dom": "^17.0.1",
     "react-g-analytics": "0.4.2",
     "react-router-dom": "^5.2.0",
+    "react-spring": "next",
```

Comment:
I guess you want to experiment with this library, but this is a huge library for a single and simple animation. Would be good to do it in pure CSS.

Reply:
It's the best and fast animation library for React to accomplish what I need since I don't know how to do it in pure CSS. Regarding the size, it's acceptable to me:

![image](https://user-images.githubusercontent.com/1091472/103620220-19a25d00-4f6e-11eb-893e-e7e23e3b4576.png)


Anyway, feel free to remove it if you don't like the idea here.